src/main/scala/package.scala

```diff
@@ -4,13 +4,21 @@ import ohnosequences.awstools.s3._
 
 package object rna16s {
 
-  val dbName = "ohnosequences.db.rna16s"
+  val version: String =
+    "7.0"
 
-  private val metadata = generated.metadata.rna16s
+  val asPath: String => String =
+    _.replace('.', '/')
```

Comment:
Where will this be used?  If it's used with the s3Prefix below, it will also change 7.0 by 7/0.

Reply:
it shouldn't be used :) removing it.